.github/ISSUE_TEMPLATE/issue.md

```diff
@@ -1,7 +1,62 @@
 ---
 name: Issue
-about: Talk about other harbor related things
-
+about: Covers everything about the project which may require some work or investigation. It is not necessarily an issue with the code, but it creates a stating point for investigation.
 ---
+<!--
+Remember, an issue is not the place to ask questions. You can use  [Stack Overflow](https://stackoverflow.com/questions/tagged/harbor)  for that, or you may want to start a discussion in [Slack](https://cloud-native.slack.com/archives/CC1E09J6S).
+
+
+Before opening a new issue, it is highly likely that someone else asked the same question before.
+
+Please search and verify that you search in those locations for answers first:
+- [ ] https://stackoverflow.com/questions/tagged/harbor
+- [ ] https://github.com/search?q=org%3Agoharbor+YOUR-ISSUE&type=issues
+- [ ] https://cloud-native.slack.com/archives/CC1E09J6S)
+- [ ] https://goharbor.io/docs/latest/
+- [ ] https://tenforward.consulting/blog/how-to-write-a-good-bug-report
+-->
+
+# Problem Descriptions
```

Comment:
Doesn't this part overlaps the content in "bug_report", if user wants to report a problem he/she should follow "bug_report.md" right?